README.md

```diff
@@ -21,14 +21,14 @@ WSLg strives to make Linux GUI applications feel native and natural to use on Wi
 
    - [NVIDIA GPU driver for WSL](https://developer.nvidia.com/cuda/wsl)
      
-   
+---

```

Comment:
I don't think we need this change as there is no other part of this document use this divider, what do you think?